### PR TITLE
Cleanup the code base

### PR DESCRIPTION
**Pull request by CorentinGS**

### CODE REVIEW

File: pkg/card/cards.go [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package card
 
 import (
-	"example/paws-quest/pkg/models"
 	"example/paws-quest/pkg/action"
```
Comment (davidbarbi3r) [inline pkg/card/cards.go:4]: do you have a plugin or smth to reorg imports ?

File: pkg/card/cards.go [hunk 2]
```diff
@@ -53,38 +53,38 @@ var commonCards = []models.Card{
...
+			action.Draw{Amount: 1},
 		},
 	},
 	{
-		ID: 		6,
-		Name: 		"Smart fist",
-		Description:"This fist is smart",
-		Cost:		4,
-		Rarity: 	models.Common,
+		ID:          6,
```
Comment (davidbarbi3r) [inline pkg/card/cards.go:60]: same question for indentation ? I'm not used to go so is there a prettier config file that follows good practice ?

File: pkg/models/models.go [hunk 3]
```diff
@@ -62,61 +60,53 @@ type GameMapService interface {
...
 
 type Card struct {
-	ID int 
-	Name string 
-	Description string 
-	Cost int
-	Rarity Rarity
-
-	Actions []IAction
+	Name        string
```
Comment (davidbarbi3r) [inline pkg/models/models.go:104]: why change properties order ?

File: pkg/models/models.go [hunk 4]
```diff
@@ -127,32 +117,30 @@ const (
...
 type Player struct {
-	ID int 
-	Name string 
-	Email string 
-	Password string 
-	Level int 
-	Experience int 
-}
\ No newline at end of file
+	Name       string
```
Comment (davidbarbi3r) [inline pkg/models/models.go:140]: same question as above

**Review by davidbarbi3r — COMMENTED** [review]
Thank you for contributing to this future AAA game

File: pkg/card/cards.go (hunk 1 above)
Comment (CorentinGS) [reply pkg/card/cards.go:4]: yes, I use https://github.com/daixiang0/gci

File: pkg/card/cards.go (hunk 2 above)
Comment (CorentinGS) [reply pkg/card/cards.go:60]: There is the built-in go fmt tool but I suggest you to use https://github.com/mvdan/gofumpt

File: pkg/models/models.go (hunk 3 above)
Comment (CorentinGS) [reply pkg/models/models.go:104]: Using the following tool : https://pkg.go.dev/golang.org/x/tools/go/analysis/passes/fieldalignment 

The order of the fields is important to optimize the size of the struct

**Review by davidbarbi3r — APPROVED** [review]
All right, thank you mate !

**Pull request merged**